### PR TITLE
Update course-contents.md

### PR DESCRIPTION
**Pull request by swayneh**

### CODE REVIEW

File: course-contents.md [hunk 1]
```diff
@@ -37,7 +38,7 @@
    - Reading data from existing sources
    - Creating data
    - Viewing data
-   - Basic knowledge about dataset (overall stats, data types)
+   - Basic knowledge about dataset (data types)
```
Comment (lindsay-fredrick) [inline course-contents.md]:
```suggestion
   - Basic knowledge about dataset (overall stats, data types)
```

I'd keep this in here - I was planning to keep the .info() and .describe() methods in this section.

File: course-contents.md [hunk 2]
```diff
@@ -54,21 +55,18 @@
...
-
-3.3. Graphical depictions of data **(Stu)**
-   -	e.g. Matplotlib and/or Seaborn 
+3.1. Exploring data structures with pandas **(Stu)**
+    - Selecting rows and columns
+    - Grouping rows and columns
+    - Sorting data
+    - Creating columns (transform)
+    - Joining datasets
+    - Descriptive stats methods (e.g. info(), describe(), value_counts, sum(), agg(), etc.)
```
Comment (lindsay-fredrick) [inline course-contents.md]:
```suggestion
    - Descriptive stats methods (e.g. value_counts, sum(), agg(), etc.)
```

See comment above re. info and describe

Comment (lindsay-fredrick) [inline course-contents.md]: Are you planning to include mutates/assigning and 'piping?' (I'd probably call it chaining in python). I think those are the only things off the old list you don't have on your new pythonic list.

Comment (swayneh) [reply course-contents.md]: Yes, i will include that in 3.1! thanks

**Pull request merged**